### DISCUSSION TITLE
Python Function Documentation Support

### DISCUSSION THREAD

**Discussion by crosleythomas** [vercel/community#2782] [Help]
### Summary

I'm looking for additional documentation for running python functions on Vercel. I've read all the documentation I can find and cloned multiple template projects, but anything I try to do beyond an endpoint that returns "Hello World" with no dependencies throws a `500 Unknown application error occurred Runtime.Unknown` with no additional information.

I've seen this error pop up in a few other threads with no one finding a solution.

Thanks!

### Example

https://flask-alt-text-generator-beta.vercel.app/

### Steps to Reproduce

One public example if the alt-text-generator project which has this issue if you try to Deploy it.

1. Deploy project
2. Hit /generate endpoint
3. See log for `Runtime.Unknown` error

But I've seen this with every other python project

**Comment by smaeda-ks** [comment]
We need a reproducible example repo to look into this further:
https://vercel.com/guides/creating-a-minimal-reproducible-example
